packages/video_player/video_player/pubspec.yaml

```diff
@@ -21,12 +21,15 @@ flutter:
 dependencies:
   meta: ^1.0.5
   video_player_platform_interface: ^2.2.0
+
   # The design on https://flutter.dev/go/federated-plugins was to leave
   # this constraint as "any". We cannot do it right now as it fails pub publish
   # validation, so we set a ^ constraint.
   # TODO(amirh): Revisit this (either update this part in the  design or the pub tool).
   # https://github.com/flutter/flutter/issues/46264
-  video_player_web: '>=0.1.1 <2.0.0'
+  video_player_web:
+    # todo(creativecreatorormaybenot): use '>=0.1.4 <2.0.0' web dependency
```

Comment:
Note that I will update this once https://github.com/flutter/plugins/pull/3081 lands.